### PR TITLE
Add new feast boolean to contentAccess as well as advice on which iOS subscription group to offer

### PR DESCRIPTION
**Pull request by paulbrown1982**
### Why do we need this? 

There is a request by the Feast team to allow access to the app or to a free trial IAP by various cohorts of the existing Supporter base (see https://docs.google.com/presentation/d/1y6Sxe5pLiq8bUSreLiPy-CsWZHf-0DmWlFSRh3Qycks/edit).

I created this PR (1) to collaborate with Ben Briggs on what the new fields will look like so that work across both teams can run in parallel and (2) to roughly demonstrate that the code changes were not extensive, and possible without requiring access to more [SupporterProduct] data within MDAPI.

### The changes 
- Added new `feast` boolean within the `contentAccess` object
- Added a new `feastIosSubscriptionGroup` field at the top level to provide advice to the app on which subscription group (SKU) to offer (this moves logic to the backend saving logic in the app having to make the decision).

### trello card/screenshot/json/related PRs etc
https://trello.com/c/qZy2vvXn and https://trello.com/c/rPUKspl9

### CODE REVIEW

File: membership-attribute-service/app/models/Attributes.scala
Comment (rupertbates) [inline membership-attribute-service/app/models/Attributes.scala:77]: You've got `isPatronTier` in here twice

**Comment by rupertbates** [comment]
Can you add an explanation of what all this means into the description please?

**Comment by paulbrown1982** [comment]
> Can you add an explanation of what all this means into the description please?

Done @rupertbates. This PR is an RFC on the approach rather than a production-ready change.

File: membership-attribute-service/app/models/Attributes.scala
Comment (rupertbates) [inline membership-attribute-service/app/models/Attributes.scala:95]: What is a subscription group?

Comment (paulbrown1982) [reply membership-attribute-service/app/models/Attributes.scala:95]: image

**Review by rupertbates — APPROVED** [review]
This should be good to go out now pending final decision on whether we need to treat RCs differently based on acquisition data

File: membership-attribute-service/test/controllers/AttributeControllerTest.scala
Comment (paulbrown1982) [inline membership-attribute-service/test/controllers/AttributeControllerTest.scala:268]: If you're gonna repair the two 'l's at the end of Successful you may as well also add the additional 'f' at the end of 'OneOf' or call it 'OneTime'!

**Pull request merged**
